lastuser_ui/templates/account.html.jinja2

```diff
@@ -38,6 +38,7 @@
           {% for extid in current_auth.user.externalids %}
             <tr>
               <td><i class="fa fa-fw fa-{{ extid.service }}"></i> {{ extid.username }}</td>
+              <td>{% trans %}Last used{% endtrans %} {{ extid.last_used_at|age }}</td>
```

Comment:
Is this `age` filter built into jinja2? can't seem to find a reference for it.

Reply:
It's in Baseframe.